zingocli/src/lib.rs

```diff
@@ -30,31 +49,36 @@ pub fn build_clap_app() -> clap::ArgMatches {
                 .action(clap::ArgAction::SetTrue))
             .arg(Arg::new("chain")
                 .long("chain").short('c')
-                .help(r#"What chain to expect, if it's not inferable from the server URI. One of "mainnet", "testnet", or "regtest""#))
-            .arg(Arg::new("from")
-                .short('f')
-                .short_alias('s')
-                .long("from")
-                .alias("seed")
-                .alias("viewing-key")
-                .value_name("from")
-                .value_parser(parse_seed)
-                .help("Create a new wallet with the given key. Can be a 24-word seed phrase or a viewkey. Will fail if wallet already exists"))
+                .help(r#"What chain to expect, if it's not inferrable from the server URI. One of "mainnet", "testnet", or "regtest""#))
+            .arg(Arg::new("seed-phrase")
+                .long("seed-phrase")
+                .hide(true)
+                .env("SEED_PHRASE")
+                .value_parser(WalletBase::parse_input_to_phrase)
+                .requires("birthday")
+                .help("Create a new wallet with the given seed phrase. Cannot be used with --view-key, or --wallet-dir."))
+            .arg(Arg::new("view-key")
```

Comment:
This seems to be a reversion of https://github.com/zingolabs/zingolib/pull/426, I'm assuming this is intended?

Reply:
Yes.   `alias`s don't support multiple parsers, and I wanted to have a separate parser for each of the view key, and seedphrase.

I left from_string in, because it's `pub`, and I wasn't sure about consumers, but @fluidvanadium says that there probably aren't third party consumers, so we can remove it and simplify the code next time we update the semver.

I removed the `from` flag because of the Zen of Python (explicit is better than implicit).   Now the args are `--seed-phrase` and `--view-key`.    For convenience and back-compatibility they are exposed as command line options, but we *could* force the user to not use them (because the environment variable path is available).  For now the user can decide between a command line option and an environment variable.